environments/prod/prod.tfvars

```diff
@@ -328,6 +328,106 @@ frontends = [
       },
     ]
   },
+  {
+      product          = "div"
+      name             = "div-amend-pfe"
```

Comment:
I'm not sure whether this needs to match anything or not.

Reply:
Should we be updating the existing entry for div-pfe then? Or would that affect the existing FD config for www.apply-divorce.service.gov.uk ?

Reply:
I think that name is fine but make sure you add entry here for shuttering as per parsing rules https://github.com/hmcts/azure-shutter-pages/blob/master/azure-pipelines.yml